mhs/common/mhs_common/state/work_description.py

```diff
@@ -81,9 +100,6 @@ def create_new_work_description(persistence_store: pa.PersistenceAdaptor,
     if not key:
         logger.error('005', 'Failed to build new work description, key should not be null or empty')
         raise ValueError('Expected key to not be None or empty')
-    if status is None:
```

Comment:
Could we not make this check whether inbound _and_ outbound status are null? Presumably we need at least one of them?

Reply:
Added the test back in as well